### PR TITLE
Publish 2023-03-02-do-you-speak-prestashop-february-2023.md

### PR DESCRIPTION
**Pull request by l-delin**
Hello,

here is February's Crowdin Monthly :-)

### CODE REVIEW

**Comment by l-delin** [comment]
ping @coffrep & @kpodemski

File: content/news/translation/2023/2023-03-02-do-you-speak-prestashop-february-2023.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+aliases: ["/news/do-you-speak-prestashop-february-2023"]
+title: "Do you speak PrestaShop? – February 2023 edition"
+subtitle: "Where you'll learn that the top translators page is back"
+date: 2023-03-06
```
Comment (kpodemski) [inline content/news/translation/2023/2023-03-02-do-you-speak-prestashop-february-2023.md]:
```suggestion
date: 2023-03-03
```

we need to publish it today, there is something else planned for Monday

Comment (l-delin) [reply content/news/translation/2023/2023-03-02-do-you-speak-prestashop-february-2023.md]: That's perfect then, thanks

**Review by coffrep — APPROVED** [review]
Great post, thanks a lot, I'm approving!

**Comment by kpodemski** [comment]
thanks @l-delin

**Pull request merged**